app/src/main/res/values-es/strings.xml

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="large_app_name">Aplicación Suscriptores GitHub</string>
-    <string name="home_progress_dialog">Obteniendo usuarios por favor espere</string>
+    <string name="subscriberlist_progress_dialog">Obteniendo detalles del suscritor, por favor espere</string>
```

Comment:
Shouldn't it be related to Subscriber List? You're duplicating the same copy below. Also in Spanish is "suscriptor".